packages/tsdocs/README.md

```diff
@@ -0,0 +1,58 @@
+# @loopback/tsdocs
+
+This module provides [API docs](https://github.com/Microsoft/tsdoc) generation
+for `@loopback/*` packages.
+
+It's built on top of https://api-extractor.com/:
+
+- https://github.com/Microsoft/web-build-tools/tree/master/apps/api-extractor
+- https://github.com/Microsoft/web-build-tools/tree/master/apps/api-documenter
+
+## Basic Use
+
+### Build api reports and doc models
+
+```sh
+npm start
+```
+
+The command above will traverse all TypeScript packages in the monorepo and run
+`api-extractor` to generate api reports and doc models into
+`loopback-next/docs/apidocs`:
+
+- `reports`: api reports
+- `reports-temp`: temporary api reports
```

Comment:
What is the purpose of these temporary reports?

Reply:
It's used for diffing api changes against `reports`.